recipe/meta.yaml

```diff
@@ -1,17 +1,15 @@
 # Full credit goes to https://github.com/conda/conda-recipes for providing this recipe.
 # It has been subsequently adapted for automated building with conda-forge.
 
-{% set version = "3.4.3" %}
+{% set version = "3.5.0" %}
 
 package:
   name: matplotlib-suite
   version: {{ version }}
 
 source:
   url: https://github.com/matplotlib/matplotlib/archive/v{{ version }}.tar.gz
-  sha256: 474c4c5555476eb35f44cfd00a4c9b7b24fff9cef685ec80e52260cc33558b0c
-  # patches:                    # [osx]
-  #   - fix_osx_1011_dep.patch  # [osx]
+  sha256: fc46f0eb54d9d51ddf0637e383afce2fdb6e352d894a81993b25d8df41eddaaf
   patches:
     - pypi.patch  # [python_impl == "pypy"]
     - pre-7.3.6-0001.patch  # [python_impl == "pypy" and win]
```

Comment:
It looks like these were removed in the Conda-Forge recipe and are only for PyPy. Do we still need them?

Reply:
I removed them